src/rules/pydocstyle/rules.rs

```diff
@@ -785,6 +786,34 @@ pub fn ends_with_period(checker: &mut Checker, docstring: &Docstring) {
     }
 }
 
+static MOOD: Lazy<Mood> = Lazy::new(Mood::new);
+
+/// D401
+pub fn use_imperative_mood(checker: &mut Checker, docstring: &Docstring) {
+    let body = docstring.body;
+
+    if let Some(index) = logical_line(body) {
+        let line = match body.lines().nth(index) {
+            Some(line) => line.trim(),
+            None => return,
+        };
+        let first_word_norm = match line.split_whitespace().next() {
+            Some(word) => normalize_word(word),
+            None => return,
+        };
+        if first_word_norm.is_empty() {
+            return;
+        }
+        if let Some(false) = MOOD.is_imperative(&first_word_norm) {
+            let diagnostic = Diagnostic::new(
+                violations::UseImperativeMood(line.to_string()),
+                Range::from_located(docstring.expr),
+            );
+            checker.diagnostics.push(diagnostic);
+        }
+    }
+}
+
```

Comment:
I think it would make sense to move this new code (along with the violation defintion) to a new file `pydocstyle/non_imperative_mood.rs` since we actually want to break up such `rules.rs` files into smaller files.

Reply:
Should I maybe make a separate PR that breaks up `pydocstyle/rules.rs` to begin with?

Reply:
You don't need to break up all of `pydocstyle/rules.rs` to introduce a new pydocstyle rule but yes if you want to break that file up you could do so in a separate PR.

Reply:
@not-my-profile 👍  Done in https://github.com/charliermarsh/ruff/pull/2003.